src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,39 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithConfirmation extends Popup {
+  constructor({ popupSelector }) {
+    const popupElement = document.querySelector(popupSelector);
+    if (!popupElement) {
+      throw new Error(`Element with selector "${popupSelector}" not found.`);
+    }
+    super({ popupSelector: popupSelector });
+    this._popupform = popupElement.querySelector(".modal__form");

```

Comment:
`_popupform`

Every variable in the project should be in `camelCase` 


`_popupForm`